monai/apps/utils.py

```diff
@@ -27,6 +28,8 @@
 from urllib.parse import urlparse
 from urllib.request import urlopen, urlretrieve
 
+import requests
```

Comment:
Regarding the ci error here: https://github.com/Project-MONAI/MONAI/actions/runs/12338243392/job/34432967743?pr=8216
Please consider using `optional_import` here and 
https://github.com/Project-MONAI/MONAI/blob/21920a34bc00a114e430e1943e1fd1f572880919/monai/bundle/scripts.py#L61

And skip in the related tests.
https://github.com/Project-MONAI/MONAI/blob/21920a34bc00a114e430e1943e1fd1f572880919/tests/test_bundle_get_data.py#L46

Please also include related test! Thanks again.